crates/sqlexec/src/remote/client.rs

```diff
@@ -193,14 +186,33 @@ impl RemoteClient {
         }
 
         let mut dst: Endpoint = dst.try_into()?;
-        if let Some(tls_conf) = tls_conf {
-            metadata.insert(CA_CERT_PATH, tls_conf.ca_cert_path.parse()?);
-            metadata.insert(DOMAIN, tls_conf.domain.parse()?);
-            let ca = std::fs::read_to_string(tls_conf.ca_cert_path)?;
+
+        let mut api_url = "https://console.glaredb.com/api/internal/authenticate/client";
+        let host = dst.uri().host().expect("invalid host");
```

Comment:
```suggestion
        let host = dst.uri().host()?;
```

To not panic here

Reply:
FYI: I merged without this change, because it required updating error/signature. If I was more familiar with Rust and the cb I would've been more able to make that change.